templates/partials/ensure-no-defaults.dockerfile.j2

```diff
@@ -0,0 +1,6 @@
+{# This partial file is used to ensure there are no packages from the defaults conda channel #}
+
+{# Check for defaults packages in rapids env #}
+RUN source activate rapids \
```

Comment:
Should we output `conda list` before the check so if it fails we can see what packages were in `defaults`?